libs/docs/core/multi-input/examples/multi-input-example/multi-input-example.component.ts

```diff
@@ -32,4 +33,8 @@ export class MultiInputExampleComponent {
     searchTermChange(text: string): void {
         console.log(text);
     }
+
+    customFilter(currentArray: (string | any)[], searchTerm: string): any {
```

Comment:
update return type

Reply:
> Type of filterFn is wrong on multi-input.component (this['dropdownValues']) as it contains a package private type. This is why in the example I used anys

Reply:
I don't think we want to include an example of a deliberately broken filter function in our docs